rmf_building_map_tools/building_crowdsim/config/configfile_generator.py

```diff
@@ -73,7 +73,9 @@ def generate_scene_file(self, output_dir):
             for item in self.crowd_sim_yaml['agent_groups']:
                 cur_group = AgentGroup()
                 cur_group.load_from_yaml(item)
-                self.scene_file.sub_elements.append(cur_group)
+                cur_generator = cur_group.sub_elements[-1]
+                if len(cur_generator.sub_elements) != 0:
```

Comment:
After a quick look. is it possible for us to use [this](https://github.com/open-rmf/rmf_traffic_editor/blob/8444941cf1633747eab49e0b2cbd9cd8d299627f/rmf_building_map_tools/building_crowdsim/config/scene_file.py#L157) `AgentGenerator.is_valid()` function?

Reply:
good idea! thanks for the suggestion